src/libraries/Common/src/System/Security/Cryptography/SlhDsa.Create.OpenSsl.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+
+namespace System.Security.Cryptography
```

Comment:
These .Create files are wrong (they match the older pattern for RSA/ECDSA/etc).  Go back to what you had before, match ML-DSA.